src/NuGet.Services.Staging.Manager/StageService/IStageService.cs

```diff
@@ -67,5 +67,11 @@ public interface IStageService
         /// Get progress report for this commit.
         /// </summary>
         BatchPushProgressReport GetCommitProgress(StageCommit commit);
+
+
+        /// <summary>
+        /// Adds a package to the stage
+        /// </summary>

```

Comment:
fixed

---
In reply to: [62967716](https://github.com/NuGet/NuGet.Staging/pull/44#discussion_r62967716) [](ancestors = 62967716)